PX.Survey.Ext/SurveyMaint.cs

```diff
@@ -1,37 +1,48 @@
 ﻿using System;
 using System.Collections;
+using System.Collections.Generic;
 using System.Linq;
+using System.Reflection;
 using PX.Common;
 using PX.Data;
+using PX.Data.Access.ActiveDirectory;
 using PX.Data.BQL;
 using PX.Data.BQL.Fluent;
 using PX.Objects.CR;
 using PX.Objects.CS;
+using PX.SM;
+using PX.Survey.Ext;
 
-namespace PX.Survey.Ext
+namespace AcumaticaSurveysLibr
 {
+    [PXCacheName("Filter users roles")]
+
     public class SurveyMaint : PXGraph<SurveyMaint, Survey>
     {
         public SelectFrom<Survey>.View SurveyCurrent;
 
+        public PXFilter<FilterUserRoles> FilterRoles;
+
+        public PXSelect<UsersInRoles, Where<UsersInRoles.applicationName, Equal<Current<PX.SM.Roles.applicationName>>,
+            And<UsersInRoles.rolename, Equal<Current<PX.SM.Roles.rolename>>>>> UsersByRole;
+
         [PXViewName(PX.Objects.CR.Messages.Attributes)]
         public CSAttributeGroupList<Survey.surveyID, SurveyCollector> Mapping;
 
         public SelectFrom<SurveyUser>.Where<SurveyUser.surveyID.IsEqual<Survey.surveyID.FromCurrent>>.View SurveyUsers;
 
-        [PXHidden]
-        [PXCopyPasteHiddenView]
-        public PXSetup<SurveySetup> SurveySetup;
+        [PXHidden] [PXCopyPasteHiddenView] public PXSetup<SurveySetup> SurveySetup;
```

Comment:
Is this formating in a single line by mistake?

Reply:
Kind of strange, as in Visual Studio I see multiple lines.

Reply:
![image](https://user-images.githubusercontent.com/14317253/96173215-39f2ea80-0f30-11eb-84e9-dcc96d7a9241.png)